widgetry/src/style/mod.rs

```diff
@@ -32,6 +32,9 @@ pub type OutlineStyle = (f64, Color);
 
 static DEFAULT_OUTLINE_THICKNESS: f64 = 2.0;
 
+// This is #EE702E, called "ab_orange_1" in Figma
```

Comment:
We still can't make `Color::hex` be `const` :(

Reply:
loooong overdue to make some of these constants.